app/src/main/java/edu/gcc/whiletrue/pingit/FAQPageFragment.java

```diff
@@ -192,37 +192,39 @@ protected Integer doInBackground(String... params) {
 
         @Override
         protected void onPostExecute(Integer errorCode) {
-            if (errorCode == 0) { //Populate the pings list if everything is clear
-                ArrayList<FAQ> faqData = new ArrayList<FAQ>();
-
-                for (int i = 0; i < categoryList.size(); i++) {
-                            try {
-                                ArrayList<ArrayList<String>> questionArr = new ArrayList<ArrayList<String>>();
-                                for(int j = 0; j< questionsList.get(i).size(); j++){
-                                    questionArr.add(new ArrayList<String>());
-                                    questionArr.get(j).add(questionsList.get(i).get(j).getString("Text"));
-                                    questionArr.get(j).add(questionsList.get(i).get(j).getString("AnswerText"));
-                                }
-
-                        //Populate each Ping object
-                        //Make sure that the category isn't empty
-                        if(questionArr.size()>0)
-                            faqData.add(new FAQ(categoryList.get(i).getString("Text"), questionArr));
-                    }catch(Exception e){}
-                    //TODO figure out why this is throwing an error when the user has no pings
-                    //something to do with Date.getTime() being called on a null object reference
+            if (getContext() != null) {
```

Comment:
Can't use getContext(), the linter won't allow it because it's only intended for use on Marshmallow and up. Instead use the context property of the GetFAQTask class. (line 169)